internal/cmds/execute-template/main.go

```diff
@@ -128,6 +128,16 @@ func run() error {
 	funcMap["replaceAllRegex"] = func(expr, repl, s string) string {
 		return regexp.MustCompile(expr).ReplaceAllString(s, repl)
 	}
+	funcMap["toSentenceCaseCommitMessages"] = func(s string) string {
+		re := regexp.MustCompile(`(?m)([[:alpha:]]+:) +([[:alpha:]])`)
+		return re.ReplaceAllStringFunc(s, func(m string) string {
+			submatches := re.FindStringSubmatch(m)
+			if len(submatches) != 3 {
+				return m
+			}
+			return fmt.Sprintf("%s %s", strings.ToLower(submatches[1]), strings.ToUpper(submatches[2]))
```

Comment:
This _will_ break sentences that begin with initialisms (e.g., `HTML generation broken` will become `Html generation broken`).

Reply:
I don't believe that is the case. I am only capturing the first letter of the commit message and converting that to uppercase.

You can confirm this for yourself by building the site and navigating to commit `23de447` under version `2.56.0`:

* [23de447](https://github.com/twpayne/chezmoi/commit/23de4476284bb694372eb378d4292a9ad33135f1) fix: YAML parsing error by escaping | in title

If I change it to `strings.ToLower(submatches[2])`:

* [23de447](https://github.com/twpayne/chezmoi/commit/23de4476284bb694372eb378d4292a9ad33135f1) fix: yAML parsing error by escaping | in title

Of course, it does not help in cases where a message contains a lowercase initialism (e.g. `html generation broken` will only become `Html generation broken`), but that is non-trivial to solve.

Reply:
You're right. I misread the regex (too much escaping required in Go regexes).